### ISSUE TITLE
Rear of Rack is not shown

### ISSUE THREAD

**Issue by nikor30** [netbox-community/netbox#6633]
### NetBox version

2.11.6

### Python version

3.8

### Steps to Reproduce

{"error": "14", "exception": "KeyError", "netbox_version": "2.11.6", "python_version": "3.8.5"}
![image](https://user-images.githubusercontent.com/4319444/122723127-dfeeb580-d272-11eb-8965-99aad1e4c81e.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/4319444/122723165-ec730e00-d272-11eb-9ac0-3023c9f7f9d1.png)

### Observed Behavior

So far thats the first rack I discovered who has this issue.

**Comment by nikor30** [comment]
![image](https://user-images.githubusercontent.com/4319444/122724031-db76cc80-d273-11eb-84a6-89b3ec3ecd24.png)
So there devices
![image](https://user-images.githubusercontent.com/4319444/122724446-37d9ec00-d274-11eb-8637-4e01afdbc6c0.png)

**Comment by nikor30** [comment]
![image](https://user-images.githubusercontent.com/4319444/122725304-22b18d00-d275-11eb-91e1-23fb4184e12c.png)

**Comment by nikor30** [comment]
Seems that it´s the only rack which has this topic.

So can you guide me where trouble shoot so maybe it´s a database issue.

Regards Niko

**Comment by jeremystretch** [comment]
Please go back and rewrite your initial post to provide all the information required by the bug report template.

**Comment by nikor30** [comment]
Hi so, I was able to get rid of the issue.
Basically some users where able to put devices into the front of the rack which have full depth and also some devices into the rear.

Maybe happened when some device types were changed to full depth.

Which the cause the rack rear not be able to be rendered.

Now I created a second rack and moved all devices from the rear to the other one which resolved the issue.
